lib/bindings/src/breez_liquid_sdk.udl

```diff
@@ -81,6 +83,21 @@ dictionary ReceivePaymentResponse {
     string invoice;
 };
 
+dictionary PrepareOnchainRequest {
```

Comment:
Why do we need the payment_type field here? Is it so we will be able to use the same API for both swap in/ swap out? I see the rational but I think it will be too confusing for users.
Also from consistency with the sdk standpoint we probably need:
1. pay_onchain
2. receive_onchain

Unless you think this is something needs to be changed?